ansible/roles/ami-ovf/tasks/main.yml

```diff
@@ -28,6 +28,22 @@
       when: ansible_virtualization_type == "virtualbox"
       command: swapon -a
 
+    - name: OVF                        | Create /.pmm-ovf file
+      when: ansible_virtualization_type == "virtualbox"
+      file:
+        path: /.pmm-ovf
+        owner: root
+        group: root
+        mode: 0600
+
+    - name: AMI                        | Create /.pmm-ami file
+      when: ansible_virtualization_type != "virtualbox"
```

Comment:
Not VirtualBox

https://github.com/Percona-Lab/percona-images/blob/b1518d0087cef704e5f3549c26df82d5b3f286ed/ansible/pmm/main.yml#L112-L131